devicemodel/hw/pci/wdt_i6300esb.c

```diff
@@ -41,15 +41,15 @@
 #define ESB_WDT_REBOOT	(0x01 << 5)   /* Enable reboot on timeout */
 #define ESB_WDT_RELOAD	(0x01 << 8)   /* Ping/kick dog  */
 #define	ESB_WDT_TIMEOUT	(0x01 << 9)   /* WDT timeout happened? */
-#define TIMER_TO_SECONDS(val)	(val >> 9)
+#define TIMER_TO_SECONDS(val)	(val >> 10) /* 10bits for 1s in a 1KHz Clock */
 
 /* Magic constants */
 #define ESB_UNLOCK1	0x80   /* Step 1 to unlock reset registers  */
 #define ESB_UNLOCK2	0x86   /* Step 2 to unlock reset registers  */
 
 #define WDT_TIMER_SIG			0x55AA
 #define DEFAULT_MAX_TIMER_VAL		0x000FFFFF
-#define	DEFAULT_RESET_TIMER_VAL		60
+#define	DEFAULT_RESET_TIMER_VAL		(60 << 10) /* timer value of 60s */
```

Comment:
better to tell the unit too

Reply:
/* stage timer set to 60s, left shift 10 bits to get the preload value in a 1KHz CLK prescaler */
#define	DEFAULT_RESET_TIMER_VAL		(60 << 10)

Reply:
THe original bug comes from that we mixed the unit. We should clarify those unit for each MACROs.

Reply:
/* stage timer is set to 60s, left shift 10 bits to transform to the preload value
 * of the watchdog timer which run in 1KHz clock */
#define DEFAULT_RESET_TIMER_VAL (60 << 10)